README.md

```diff
@@ -1,2 +1,64 @@
-# tw-academy-s9
-Homework repository
+# Sample Style Guide
+Technical Writing Academy - May 2021
+
+## Bold
+
+Use bold to highlight button names, field names, menus, radio buttons, or checkboxes. For example:
+* Click the **create** button to start the process.
+* Enter the IP adress in the **IP adress** field.
+
+## Italics
+
+Use italics to emphasize a word or a group of words. For example:
+* *At least two* reviews are requiered before you check in your code.
+* You must *not* enter without your ID card.
+
+## Fixed Font
+Use a fixed font to highlight:
+* **Commands or pieces of code.** For example: Run the `npm install text-cli` command to install the utiliy.
+* **File or folder names.** For example: Read the `README.md` file before you proceed.
+
+## Abbreviations
```

Comment:
This content would be an opportunity to include tables in your markdown.

Reply:
Done!